### DISCUSSION TITLE
SpotifyMultiroom

### DISCUSSION THREAD

**Discussion by KotelczukB** [B5r1oJ0A9G/teufel_raumfeld#22] [Q&A]
Hello, 
I am quite new to the topic of homeassistant and am currently trying to integrate the teufel_raumfeld integration. My problem is that I can not find a way to turn Spotify Mulitroom on and off (the spotifymode of the host). Is there a configuration that I am missing? 

Secondly, I don't understand why when I connect to a speaker via SporifyConnect, it shows as turned off.

**Comment by B5r1oJ0A9G** [comment]
Hello,
This integration does not yet offer any configuration regarding Spotify. The use of Spotify Multi-Room as well as Single-Room should be supported to some extent. See: #4
However, I can't use Spotify in my setup and therefore I am not able to test Spotify support.
